src/Dfe.PlanTech.Infrastructure.Contentful/Content/Renderers/Models/PartRenderers/HyperlinkRenderer.cs

```diff
@@ -47,4 +58,15 @@ private static void AddTagAndHref(RichTextContent content, StringBuilder stringB
         stringBuilder.Append(content.Data?.Uri ?? "");
         stringBuilder.Append('"');
     }
+
+    private static void AddBlankTarget(StringBuilder stringBuilder)
+    {
+        stringBuilder.Append("target=\"_blank\" rel=\"noopener\"");
+    }
+
+    private static bool IsExternalLink(RichTextContent content)
+    {
+        var uri = content.Data?.Uri ?? "";
+        return uri.StartsWith("http", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Is it possible for URIs to ommit the scheme, and not be relative?

E.g. we have relative links (e.g. `/content/hello-world`), and we have full URIs (e.g. `https://blabla.com`). Does Contentful allow, say `google.com` or `www.google.com`?

Reply:
If we omit the scheme it'll treat it as an internal url so you'd get
`https://localhost:8080/google.com` or `https://localhost:8080/www.google.com`
so luckily I think this is not an issue